3rdparty/dxc/CMakeLists.txt

```diff
@@ -108,8 +126,8 @@ if(_NBL_ENABLE_DXC_COMPILE_TESTS_)
 	add_subdirectory(CT CT)
 endif()
 
-include("${DXC_BUILD_DIR}/nbl/generate/dxc/config/debug.cmake")
-include("${DXC_BUILD_DIR}/nbl/generate/dxc/config/release.cmake")
+# include("${DXC_BUILD_DIR}/nbl/generate/dxc/config/debug.cmake")
```

Comment:
don't comment this out, we generate on fly information about where the `dxcompiler` target will output it's archive and dynamic library